### ISSUE TITLE
Compile fail mvn clean install -Dskip-validate-sources 

### ISSUE THREAD

**Issue by skabashnyuk** [eclipse-che/che#7744]
### Reproduction Steps
```
ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /Users/sj/dev/src/redhat/che/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ConfigureClasspath.java:[109,8] error: reference to until is ambiguous
  both method until(Predicate<T>) in FluentWait and method <V>until(Function<? super T,V>) in FluentWait match
  where T,V are type-variables:
    T extends Object declared in class FluentWait
    V extends Object declared in method <V>until(Function<? super T,V>)
[ERROR] /Users/sj/dev/src/redhat/che/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ConfigureClasspath.java:[119,8] error: reference to until is ambiguous
  both method until(Predicate<T>) in FluentWait and method <V>until(Function<? super T,V>) in FluentWait match
  where T,V are type-variables:
    T extends Object declared in class FluentWait
    V extends Object declared in method <V>until(Function<? super T,V>)
[ERROR] /Users/sj/dev/src/redhat/che/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ConfigureClasspath.java:[163,8] error: reference to until is ambiguous
  both method until(Predicate<T>) in FluentWait and method <V>until(Function<? super T,V>) in FluentWait match
  where T,V are type-variables:
    T extends Object declared in class FluentWait
    V extends Object declared in method <V>until(Function<? super T,V>)
[ERROR] /Users/sj/dev/src/redhat/che/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ConfigureClasspath.java:[173,8] error: reference to until is ambiguous
```

**Comment by skabashnyuk** [comment]
Caused by guava dependency taken from errorprone
```
[DEBUG] Dependency collection stats: {ConflictMarker.analyzeTime=0, ConflictMarker.markTime=0, ConflictMarker.nodeCount=159, ConflictIdSorter.graphTime=0, ConflictIdSorter.topsortTime=0, ConflictIdSorter.conflictIdCount=60, ConflictIdSorter.conflictIdCycleCount=0, ConflictResolver.totalTime=0, ConflictResolver.conflictItemCount=99, DefaultDependencyCollector.collectTime=18, DefaultDependencyCollector.transformTime=0}
[DEBUG] org.apache.maven.plugins:maven-compiler-plugin:jar:3.7.0:
[DEBUG]    com.google.errorprone:error_prone_core:jar:2.1.1:runtime
[DEBUG]       com.google.errorprone:error_prone_annotation:jar:2.1.1:runtime
[DEBUG]       com.google.errorprone:error_prone_check_api:jar:2.1.1:runtime
[DEBUG]          com.googlecode.java-diff-utils:diffutils:jar:1.3.0:runtime
[DEBUG]          com.github.kevinstern:software-and-algorithms:jar:1.0:runtime
[DEBUG]       com.github.stephenc.jcip:jcip-annotations:jar:1.0-1:runtime
[DEBUG]       org.pcollections:pcollections:jar:2.1.2:runtime
[DEBUG]       com.google.guava:guava:jar:21.0:runtime
[DEBUG]       com.google.auto:auto-common:jar:0.7:runtime
[DEBUG]       com.google.code.findbugs:jFormatString:jar:3.0.0:runtime
[DEBUG]       com.google.code.findbugs:jsr305:jar:3.0.0:runtime
[DEBUG]       org.checkerframework:dataflow:jar:2.1.14:runtime
[DEBUG]          org.checkerframework:javacutil:jar:2.1.14:runtime
[DEBUG]       com.google.errorprone:javac:jar:9-dev-r4023-3:runtime
[DEBUG]       com.google.errorprone:error_prone_annotations:jar:2.1.1:runtime
[DEBUG]    org.codehaus.plexus:plexus-compiler-javac-errorprone:jar:2.5:runtime
[DEBUG]       org.codehaus.plexus:plexus-utils:jar:3.0.10:compile
```

**Comment by skabashnyuk** [comment]
Fixed by reverting guava to 20.0 since selenium a not compatible with 21 or above.
